index.html

```diff
@@ -1,47 +1,48 @@
 <!doctype html>
 
 <html lang="en">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
 <head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link rel="stylesheet" type="text/css" href="application.css">
 </head>
+
 <body>
-  <div class= "master-container">
-    <a href= "book-list.html" class = "return-button"> Atrás </a>
-    <div class= "cover-text-container">
-      <img class= "badge"
-      src= "https://trello-attachments.s3.amazonaws.com/607457e8ad4162814da41ef0/607457e8ad4162814da41f19/x/3b4483825e105c5d0ae010a89df4d9b3/badge.png"
-      alt= "Cerulean and earls green badge with a star in the center"
-      >
-      <img class= "cover-page" 
-      src= "https://trello-attachments.s3.amazonaws.com/607457e8ad4162814da41ef0/607457e8ad4162814da41f17/x/6554b96f839abbe659429a9f8c79115d/book-cover.png" 
-      alt= "Wolox books cover page"
-      >
-      <div class= "container-text">
-        <h1 class= "book-title">
-          <strong>Título del libro</strong> 
-          <p class= "title-genre"> (género)</p>
-        </h1>
-        <div class= "title-border"></div>
-        <div class="container-secondary-text">
-            <h2 class="secondary-title">
-            <strong>Autor del libro:</strong> 
-            <p class= "secondary-detail"> Nombre del autor del libro</p> 
-            </h2>
-            <h2 class="secondary-title">
-            <strong>Editorial:</strong>
-            <p class= "secondary-detail"> Nombre de la editorial</p>
-            </h2>
-            <h2 class="secondary-title">
-            <strong>Año de publicación:</strong>
-            <p class= "secondary-detail"> Año de publicación</p>
-            </h2>
+  <div class="index-master-container">
+    <a href="book-list.html" class="return-button"> Atrás </a>
+    <div class="cover-text-container">
+      <div class="cover-page-badge-container">
+        <img class="cover-page" src="Images/book-cover.png" alt="Wolox books cover page">
+        <img class="badge" src="Images/badge.png" alt="Cerulean and earls green badge with a star in the center">
+      </div>
+      <p class="book-title">
+        Título del libro
+      </p>
+      <p class="book-genre"> (género)</p>
+      <div class="container-text">
+        <div class="container-book">
```

Comment:
Is this element a line that separates sections?

Reply:
yes, ive realized its just a line that separates sections, i think i could include the border inside the detail container